packages/core/src/routes/connector.ts

```diff
@@ -133,6 +136,17 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         })
       );
 
+      if (body.metadata?.target && connectorFactory.type === ConnectorType.Social) {
+        const connectors = await getLogtoConnectors();
+        assertThat(
+          !connectors.some(
+            ({ metadata: { target, platform } }) =>
+              target === body.metadata?.target && platform === connectorFactory.metadata.platform
```

Comment:
should we throw earlier if the target is missing in the body?

Reply:
We are not doing this check if `target` is not presented in `body.metadata` (this use case is VALID). See the entrance condition of this outer layer if statement.